docs/preview/03-Features/correlation.md

```diff
@@ -21,65 +21,44 @@ Three kind of correlation ID's are used to create the relationship:
+3. When a call is made to service B, the **transaction ID** is sent but also the **operation parent ID** in the form of a W3C structure: `00-transactionId-parentId-00`.
+4. The `def` part of this ID, describes the new parent ID for service B (when service B calls service C, then it will use a different parent ID)
 5. Service B responds to service A with the same information as the call to service B.
-6. The user receives both the **transaction ID** and **operation parent ID** in their final response.
+6. The user receives both the **transaction ID** and **operation ID** in their final response.
 
-💡 This correlation is based on the `RequestId` and `X-Transaction-ID` HTTP request/response headers, however, you can fully configure different headers in case you need to.
-💡 The `X-Transaction-ID`  can be overridden by the request, meaning: if the HTTP request already contains a `X-Transaction-ID` header, the same header+value will be used in the HTTP response.
+💡 This correlation is based on the `traceparent` HTTP request/response header, however.
 
 Additional [configuration](#configuration) is available to tweak this functionality.
 
-## Installation
+### ⚡ Automatic dependency tracking
+When choosing for the W3C HTTP correlation format, Arcus and Microsoft technology works seemingly together. When a HTTP request is received on a service that uses the Arcus W3C HTTP correlation, all remote dependencies managed my Microsoft (HTTP, ServiceBus, EventHubs...) are tracked automatically, without additional configuration.
 
+## Installation
 This feature requires to install our NuGet package:
 
 ```shell
 PM > Install-Package Arcus.WebApi.Logging
 ```
 
 ## Usage
-
 To fully benefit from the Arcus' HTTP correlation functionality, both sending and receiving HTTP endpoints should be configured.
 
 ### Sending side
-
-To make sure the correlation is added to the HTTP request, following additions have to be made:
+To make sure the correlation is added to the HTTP request, following additions have to be made.
 
 ```csharp
 using Microsoft.AspNetCore.Builder;
 
 WebApplication builder = WebApplication.CreateBuilder();
 builder.Services.AddHttpCorrelation();
-builder.Services.AddHttpClient("from-service-a-to-service-b")
-                .WithHttpCorrelationTracking();
+builder.Services.AddHttpClient("from-service-a-to-service-b");
 
 WebApplication app = builder.Build();
 ```
 
 Then, the [created `HttpClient`](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests) will be enhanced with HTTP correlation tracking. This means that the request will be altered to include the HTTP correlation information and the endpoint will be tracked as a [HTTP dependency telemetry](https://observability.arcus-azure.net/Features/writing-different-telemetry-types#measuring-http-dependencies).
```

Comment:
Is this still true ?  The `WithCorrelationTracking` extension is no longer called, so how will the request be altered ?

Reply:
Because of the auto-dependency tracking system. It uses activities behind the scenes, where the `HttpClient` will send diagnostic messages to.

Reply:
I'm wondering if we should keep that paragraph.  I'm not sure if it brings some added value to the consumer ?

Reply:
Yeah, maybe this is rather obvious. I'll remove it 👍

Reply:
Removed here: https://github.com/arcus-azure/arcus.webapi/pull/392/commits/73e2c417fadc87af5ee0ede627b914d6f183d9de